checkov/terraform/context_parsers/parsers/provider_context_parser.py

```diff
@@ -24,11 +24,14 @@ def _is_block_signature(self, line_num: int, line_tokens: List[str], entity_cont
 
         end_line = self._compute_definition_end_line(line_num)
         provider_type = entity_context_path[0]
-        provider_obj = hcl2.loads(
-            "\n".join(
-                map(lambda obj: obj[1], self.file_lines[line_num - 1 : end_line if end_line > line_num else line_num])
-            )
-        )["provider"][0]
+        try:
+            provider_obj = hcl2.loads(
+                "\n".join(
+                    map(lambda obj: obj[1], self.file_lines[line_num - 1 : end_line if end_line > line_num else line_num])
+                )
+            )["provider"][0]
+        except Exception:
```

Comment:
I think it would be good to log here something, when it fails, either info or debug level